docs/advanced/Testing.md

```diff
@@ -1,9 +1,35 @@
 # Examples of Testing Sagas
 
-**WIP**
+There is ongoing discussion in [this issue](https://github.com/redux-saga/redux-saga/issues/518) about the best way to test Sagas.
 
-See Repository Examples:
+
+In the repo we also have two examples showing how Sagas can be tested:
 
 https://github.com/redux-saga/redux-saga/blob/master/examples/counter/test/sagas.js
 
 https://github.com/redux-saga/redux-saga/blob/master/examples/shopping-cart/test/sagas.js
+
+
+
+***External Resources***
+
+Redux Saga Test Plan - https://github.com/jfairbank/redux-saga-test-plan
+
+From the README:
```

Comment:
I dont like including the README's there for the beforementioned reasons and wouldnt like to include them in our docs as they are. Maybe we could reach to the authors for providing us a brief description (why & how) of the libraries? cc @jfairbank @stoeffel